tests/test_permutation.py

```diff
@@ -69,3 +70,15 @@ def test_permutation_inverse():
     expected = [max(perm_vector)-i for i in perm_vector]
     expected.reverse()
     assert perm.perm_vector == expected
+
+
+@pytest.mark.parametrize("perm_list", [[2, 0, 1], [2, 3, 1, 0]])
+def test_n_mode_permutation_in_2_mode_perms(perm_list):
```

Comment:
I also try a random test a thousand times, it's working perfectly ! 😀 
```
        import random
        my_perm_list = list(range(15))
        random.shuffle(my_perm_list)
        n_mode_perm = comp.PERM(my_perm_list)
```

Reply:
I loved this test, i included it. Not running a 1000 times but only 10 haha